### PR TITLE
Portfolio: Desktop version

### PR DESCRIPTION
**Pull request by mulugithub**
## Desktop version 
The following are the requirements of this project:

- Use media queries in the CSS file to make the desktop version look like the selected tmeplate (Desktop: >=768px).
- Replicate the design as much as possible (e.g., font, colors, images, text, margins) using [the info from the Figma templates](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### CODE REVIEW

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @mulugithub, ✋  

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈

giphy

## Highlights
✅ Followed GitHub flow.
✅ Descriptive PR title & summary.
✅ Milestone works as expected.

## Required Changes ♻️

- [ ] Kindly note that there is a visible horizontal bar in your project, please find the element that is responsible for this issue and fix it.

  Screenshot 2023-06-14 212117

- [ ] Kindly note that your buttons are missing the button interact "Pressed", kindly add it.
    - You can use the pseudo-class `active`.
  
  Screenshot 2023-06-13 235634



_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ofarouq310 — CHANGES_REQUESTED** [review]
Hi @mulugithub, ✋  

Good job so far! 💪 
There are some issues that you still need to work on to go to the next project but you are almost there! 🥈

giphy

## Highlights
✅ Followed GitHub flow.
✅ Descriptive PR title & summary.
✅ Milestone works as expected.

## Required Changes ♻️

- [ ] Kindly note that there is a visible horizontal bar in your project, please find the element that is responsible for this issue and fix it, check the `margins` that have been used in your project,  since the issue is caused by them.

  Screenshot 2023-06-14 212117

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -196,96 +242,103 @@ <h1 class="project-title">Multi-Post Stories</h1>
...
-          <li>
-            <div class="lang-arrow">
-              <h3 class="skill-name">Languages</h3>
-              <img
-                src="images/Union down arrow.png"
-                class="arrow"
-                alt="down arrow"
-              />
-            </div>
+        <div class="about-wrapper">
```
Comment (stanleySimeon) [inline index.html:250]: As mentioned the previous reviewer, Kindly I suggest you stisk to the design as much as you can.

Screenshot 2023-06-14 at 5 26 36 PM

**Review by stanleySimeon — CHANGES_REQUESTED** [review]
# Hi @mulugithub, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Meaningful commit message
- Good PR description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nicupop729 — APPROVED** [review]
Hello @mulugithub 👋,

_This is an additional review as a result of the second opinion request._

Great job on the excellent effort put into this project! :clap:

Since I was unable to replicate the error mentioned by the previous reviewer, I believe you are ready to move forward to the next milestone.

As this was the only reason that your project was not approved it will be approved as soon as possible.
Please note that since we will approve it manually so this could take a few hours!

Keep rocking 🚀!

**Pull request merged**